### PR TITLE
feat: add Dockerfile for Keepalived Exporter

### PR DESCRIPTION
**Pull request by clwluvw**
Add Dockerfile and docs for Keepalived containerized deployment modes

### CODE REVIEW

**Review by mehdy — CHANGES_REQUESTED** [review]
As we discussed.

File: README.md [hunk 1]
```diff
@@ -50,7 +50,23 @@ container-tmp-dir  | Keepalived container tmp volume path, defaults to `/tmp`.
...
+./keepalived-exporter --container-name keepalived --container-tmp-dir /tmp
+```
+
+### Keepalived and Keepalived Exporter on docker
+
+Volume docker socket (`/var/run/docker.sock`) to Keepalived Exporter cotnainer in the same path and pass the args like as using Keepalived on container
+
+```bash
+docker run -v keepalived-data:/tmp/ ... $KEEPALIVED_IMAGE
+docker run -v /var/run/docker.sock:/var/run/docker.sock -v keepalived-data:/tmp/keepalived-data:ro -p 9165:9165 keepalived-exporter --container-name keepalived --container-tmp-dir "/tmp/keepalived-data"
```
Comment (clwluvw) [inline README.md:67]: This needs to replace with an image registry path.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM golang:alpine
+
+RUN apk add --no-cache make git bash
+
+WORKDIR /keepalived-exporter
+
+COPY . .
```
Comment (mehdy) [inline Dockerfile:7]: I don't remember why, but `ADD` was preferred. :))

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+RUN make build
```
Comment (mehdy) [inline Dockerfile:9]: Please use multistage build to separate the building stage and the final image.

**Review by mehdy — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 3 above)
Comment (clwluvw) [reply Dockerfile:9]: Do you mind having this with #58?

**Review by mehdy — APPROVED** [review]

**Pull request merged**